internal/patches.go

```diff
@@ -253,3 +269,27 @@ func (m *manifestPatcher) ApplyPatches() ([]map[string]interface{}, error) {
 
 	return manifests, nil
 }
+
+// Initializes the in-memory file system with base directory and open API schema
+func InitializeInMemoryKustomizeDir(fSys filesys.FileSystem, schema string) (err error) {
+	err = fSys.Mkdir(kustomizeDir)
+	if err != nil {
+		return fmt.Errorf("an unexpected error occurred when configuring Kustomize: %w", err)
+	}
+
+	if schema != "" {
+		schema = filepath.Clean(schema)
+
+		schemaJSON, err := os.ReadFile(schema)
+		if err != nil {
+			return fmt.Errorf("unable to open file: %s, err: %w ", schema, err)
+		}
+
+		err = fSys.WriteFile(path.Join(kustomizeDir, localSchemaFileName), schemaJSON)
```

Comment:
Just so I'm following, this writes a file from an arbitrary place to a `schema.json` file, correct? Probably okay, but this strays from what we currently do pointing at files in the same directory, so I'll get another review on it.

Reply:
Fixed. I added a check for it and slightly modified the verifyManifestPath to indicate the right file type.